applications/ParticleMechanicsApplication/custom_conditions/grid_based_conditions/mpm_grid_base_load_condition.cpp

```diff
@@ -117,11 +117,11 @@ namespace Kratos
 
         for (unsigned int i = 0; i < number_of_nodes; i++)
         {
-            const array_1d<double, 3 > & Displacement = r_geometry[i].FastGetSolutionStepValue(DISPLACEMENT, Step);
+            const array_1d<double, 3 > & displacement = r_geometry[i].FastGetSolutionStepValue(DISPLACEMENT, Step);
```

Comment:
this class looks quite similar to StructuralMechanics, is it copy-paste?

Reply:
yes, it was last year. The application of static load is pretty much the same with FEM.

Reply:
can you use the same Condition then?

Reply:
Of course, you can do that, but this then requires the user to compile structural as well. I want to avoid that to the most extent, as the fluid version of MPM is coming soon too.

Reply:
TBH for me this is a quite weak argument, for several reasons:
This however is not the place to discuss this, also since it has been discussed several times already. Note though that in the long term we will go in that direction (e.g. the PotentialFlowApp), otherwise the code becomes unmaintainable
Esp since this is a runtime and not a compiletime dependency